pygmt/src/grdsample.py

```diff
@@ -71,6 +71,7 @@ def grdsample(
     registration : str or bool
         [**g**\|\ **p**\ ].
         Set registration to **g**\ ridline or **p**\ ixel.
+        Note: ``translate`` and ``registration`` are mutually exclusive.
```

Comment:
1. We have renamed `translate` to `toggle` in PR #4187, so please use `toggle` instead.
2. Please remove the note below the docstring of `toggle`